### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Black-Star-Pen**

### CODE REVIEW

**Review by Erwan60 — APPROVED** [review]

File: client/src/App.css [hunk 1]
```diff
@@ -48,3 +49,6 @@ h6 {
 p {
   font-weight: 400;
 }
+button {
```
Comment (Alexandre78R) [inline client/src/App.css]: add class name ?

File: client/src/components/Collaborater/collaborater.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.Collabo-container {
```
Comment (Alexandre78R) [inline client/src/components/Collaborater/collaborater.css]: Rename your css class like the other xxx-xxx without capital letters please

File: client/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -90,6 +90,12 @@ function Footer() {
           <img src={faceImage} alt="DevLog" />
         </div>
       </section>
+
+      <div className="copyright-responsive">
+        <div>
```
Comment (Alexandre78R) [inline client/src/components/Footer/Footer.jsx]: div ??  --> fragement react ?

File: client/src/components/Collaborater/Collaborater.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import "./collaborater.css";
+
+function Collaborater() {
+  return (
+    <main>
+      <section className="Collabo-container">
+        <div>
```
Comment (Alexandre78R) [inline client/src/components/Collaborater/Collaborater.jsx]: div ??  --> fragement react ? or delete div 👍

File: client/src/main.jsx [hunk 5]
```diff
@@ -34,11 +41,19 @@ const router = createBrowserRouter ([
       },
     ],
   },
+  {
+    path: "/Collabo",
```
Comment (Alexandre78R) [inline client/src/main.jsx]: collaborator rename please

Comment (Alexandre78R) [reply client/src/main.jsx]: Place this route in the children of APP

File: client/src/main.jsx [hunk 6]
```diff
@@ -34,11 +41,19 @@ const router = createBrowserRouter ([
...
+    element: <Collaborater />,
+  },
+  {
+    path: "/footer",
```
Comment (Alexandre78R) [inline client/src/main.jsx]: delete route footer ..

File: client/src/pages/Table/Table.jsx [hunk 7]
```diff
@@ -1,13 +1,14 @@
...
-       
-        <h1>table</h1>
-      </div>
-    );
-  }
-  
-  export default Table;
\ No newline at end of file
+  return (
+    <div>
```
Comment (Alexandre78R) [inline client/src/pages/Table/Table.jsx]: div ?? --> fragement react ?

File: client/src/pages/Welcome/Welcome.jsx [hunk 8]
```diff
@@ -3,46 +3,50 @@ import AutoAwesomeIcon from "@mui/icons-material/AutoAwesome";
...
-          className="signup-btn"
-          onClick={() => navigate("/Connection")}
-        >
-          Sign Up
-        </button>
-      </section>
-      <p className="font-style">DEV LOG</p>
-      <img src={logo} className="logo-devlog" alt="Logo DevLog with slogan" />
+    <>
+      <div className="welcome-page">
```
Comment (Alexandre78R) [inline client/src/pages/Welcome/Welcome.jsx:12]: replace div ---> main

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by Alexandre78R — APPROVED** [review]

**Pull request merged**